django_fido/backends.py

```diff
@@ -26,15 +27,43 @@ def is_fido_backend_used() -> bool:
     return False
 
 
-class Fido2AuthenticationBackend(object):
+class BaseFido2AuthenticationBackend(object):
```

Comment:
```suggestion
class BaseFido2AuthenticationBackend(ABC):
```

Since it's a base class I'd use ABC to mark it as such.